engine/src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -36,6 +36,9 @@ object SgeBackend {
     def script = jobDescriptor.callRootPath.resolve("script.sh")
     def returnCode = jobDescriptor.callRootPath.resolve("rc")
   }
+
+  private lazy val oldGoogleConf = GoogleConfigurationAdapter.gcloudConf.get
+  lazy val gcsConf = Try(GoogleConfiguration(oldGoogleConf.appName, oldGoogleConf.userAuthMode.getOrElse(oldGoogleConf.cromwellAuthMode)))
```

Comment:
I'm not too keen on having copy/pasted google blocks in every backend. Is it possible to pass in the generic FS ref instead of having each backend forced to support each filesystem?